### DISCUSSION TITLE
Image is all black in iced

### DISCUSSION THREAD

**Discussion by EMUNES** [hecrj/iced#1967] [Q&A]
All images can display so I assume image  paths are good, but they are all black. I  can open and see them normally in vscode, though.

All image are *.png* files and all of them are black  in iced. Anyone has any idea about this problem?

Below is my code to show the image.

```
fn view(&self) -> Element<Self::Message> {
        let card = image(String::from("asset/img/") + &self.player_hand.cards[0].get_id() + ".png");
        let card_text = text(&self.player_hand.cards[0]);
        column!(card, card_text).width(Length::Fill).height(Length::Fill).align_items(Alignment::Center).padding(40).into()
    }
```

My dependencies:

```
[dependencies]
iced = {version = "0.9.0", features = ['image']}
rand = "0.8.5"
```

**Comment by nicksenger** [comment]
This also shows up in wasm and is fixed by "skipping" the first layer in the image atlas. Despite a fair bit of tinkering I still haven't been able to figure out why though. It seems _likely_ this is an upstream wgpu issue, what are you targeting?

**Reply by EMUNES** [reply]
It's a toy project to learn and try iced. It turns out that images in iced are fine after I compile the project on Windows11 (The problem occurs on pop_os!). I have no idea why this happens...
